.travis.yml

```diff
@@ -26,9 +26,6 @@ install:
 script:
   - npm run-script coverage
 
-after_script:
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
And I enabled it in https://coveralls.io/repos/new?name=alexa-js&service=github

<img width="747" alt="screen shot 2017-02-04 at 4 25 31 pm" src="https://cloud.githubusercontent.com/assets/542335/22621653/8efca320-eaf6-11e6-96a9-c40e40ccafa2.png">